### DISCUSSION TITLE
v0.6.0

### DISCUSSION THREAD

**Discussion by JorTurFer** [kedacore/http-add-on#811] [Announcements]
### New

- **General**: Add manifests to deploy the Add-on ([#716](https://github.com/kedacore/http-add-on/issues/716))

### Improvements

- **Scaler**: Decrease memory usage by allowing increasing stream interval configuration ([#745](https://github.com/kedacore/http-add-on/pull/745))

### Fixes

- **Interceptor**: Add support for streaming responses ([#743](https://github.com/kedacore/http-add-on/issues/743))
- **Interceptor**: Fatal error: concurrent map iteration and map write ([#726](https://github.com/kedacore/http-add-on/issues/726))
- **Interceptor**: Keep original Host in the Host header ([#331](https://github.com/kedacore/http-add-on/issues/331))
- **Interceptor**: Provide graceful shutdown for http servers on SIGINT and SIGTERM ([#731](https://github.com/kedacore/http-add-on/issues/731))
- **Operator**: Remove ScaledObject `name` & `app` custom labels ([#717](https://github.com/kedacore/http-add-on/issues/717))
- **Scaler**: Provide graceful shutdown for grpc server on SIGINT and SIGTERM ([#731](https://github.com/kedacore/http-add-on/issues/731))
- **Scaler**: Reimplement custom interceptor metrics ([#718](https://github.com/kedacore/http-add-on/issues/718))

### Deprecations

You can find all deprecations in [this overview](https://github.com/kedacore/http-add-on/labels/breaking-change) and [join the discussion here](https://github.com/kedacore/http-add-on/discussions/categories/deprecations).

New deprecation(s):

- **General**: `host` field deprecated in favor of `hosts` in `HTTPScaledObject` ([#552](https://github.com/kedacore/http-add-on/issues/552))

### Other

- **General**: Adding a changelog validating script to check for formatting and order ([#761](https://github.com/kedacore/http-add-on/pull/761))
- **General**: Skip not required CI checks on PRs on new commits ([#801](https://github.com/kedacore/http-add-on/pull/801))

## New Contributors
* @luckymrwang made their first contribution in https://github.com/kedacore/http-add-on/pull/727
* @similark made their first contribution in https://github.com/kedacore/http-add-on/pull/746
* @itayariel made their first contribution in https://github.com/kedacore/http-add-on/pull/743
* @tommy351 made their first contribution in https://github.com/kedacore/http-add-on/pull/745

**Full Changelog**: https://github.com/kedacore/http-add-on/compare/v0.5.0...v0.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/http-add-on/releases/tag/v0.6.0'>v0.6.0</a>.</em>

**Comment by dbeltman** [comment]
#331 Very nice, have tested it and it works now
